cylc/rose/stem.py

```diff
@@ -451,6 +466,34 @@ def process(self):
         return self.opts
 
 
+def get_source_opt_from_args(opts, args):
+    """Convert sourcedir given as arg or implied by no arg to opts.source.
+
+    Possible outcomes:
+        No args given:
+            Install a rose-stem suite from PWD/rose-stem
+        Relative path given:
+            Install a rose-stem suite from PWD/arg
+        Absolute path given:
+            Install a rose-stem suite from specified abs path.
+
+    Returns:
+        Cylc options with source attribute added.
+    """
+    if len(args) == 0:
+        # sourcedir not given:
+        return opts
+    elif os.path.isabs(args[-1]):
+        # sourcedir given, and is abspath:
+        opts.source = args[-1]
+    else:
+        # breakpoint()
```

Comment:
```suggestion
```

Reply:
This is here to help future devs debug. There is a note in the code https://github.com/cylc/cylc-rose/blob/d669a2efb00e05f13efa544a7d8cb4d66a42adb3/tests/functional/test_rose_stem.py#L57

Reply:
That is referring to this actually
https://github.com/cylc/cylc-rose/blob/b0eb4a775927f904a4f604ee08240c09a4d51052/tests/functional/test_rose_stem.py#L229